engine/src/vega_collection_utils.cpp

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Did you mean to implement this twice?

Reply:
vega_collection_utils is now implemented entirely in the header. This .cpp file is defunct already. I should probably just remove it.

Thanks for pointing that out.

Reply:
File removed.